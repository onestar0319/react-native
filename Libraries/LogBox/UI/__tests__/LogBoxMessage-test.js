/**
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @format
 * @emails oncall+react_native
 * @flow
 */

'use strict';

const React = require('react');
const LogBoxMessage = require('../LogBoxMessage').default;
const render = require('../../../../jest/renderer');

describe('LogBoxMessage', () => {
  it('should render message', () => {
    const output = render.shallowRender(
      <LogBoxMessage
        style={{}}
        message={{
          content: 'Some kind of message',
          substitutions: [],
        }}
      />,
    );

    expect(output).toMatchSnapshot();
  });

  // TODO: Add tests for text substitutions
});
