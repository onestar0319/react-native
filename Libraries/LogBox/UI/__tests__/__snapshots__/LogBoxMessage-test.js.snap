// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogBoxMessage should render message 1`] = `
<React.Fragment>
  <Text>
    Some kind of message
  </Text>
</React.Fragment>
`;
